inst/sympref.m

```diff
@@ -420,6 +420,13 @@
 %! delete('tmp_python_cmd.py')
 
 %!test
+%! warning ('off', 'OctSymPy:sympref:invalidarg', 'local');
```

Comment:
Its pretty horrible: https://github.com/cbm755/octsympy/blob/master/misc/extract_tests_for_matlab.m

Probably easy enough to change it to output one file per test instead of one `test_...` file per .m file.  None of the Matlab stuff is very high priority for me or AFAIK anyone else.